### PR TITLE
Wrote unit tests for server layers event, event-type, image.

### PR DESCRIPTION
**Pull request by VadimBlagiy**
Wrote unit tests for server layers: event, event-type, image. 
However need else to write test cases for the save and update methods.

### CODE REVIEW

File: sandbox-web/src/test/java/com/exadel/team2/sandbox/service/impl/EventServiceImplTest.java [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(MockitoExtension.class)
+@SpringBootTest
```
Comment (ikantsevich) [inline sandbox-web/src/test/java/com/exadel/team2/sandbox/service/impl/EventServiceImplTest.java:24]: Please create a base test class with these annotations and extends all your tests from a base class

File: sandbox-web/src/test/java/com/exadel/team2/sandbox/service/impl/EventServiceImplTest.java [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+class EventServiceImplTest {
+
+    private static final long DUMMY_ID = 1L;
```
Comment (ikantsevich) [inline sandbox-web/src/test/java/com/exadel/team2/sandbox/service/impl/EventServiceImplTest.java:27]: Please rename to a more proper name. ex: EVENT_ID

File: sandbox-web/src/test/java/com/exadel/team2/sandbox/service/impl/EventServiceImplTest.java [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+    @MockBean
+    private EventDAO eventDAO;
+
+    @Autowired
+    private EventServiceImpl eventServiceImpl;
```
Comment (ikantsevich) [inline sandbox-web/src/test/java/com/exadel/team2/sandbox/service/impl/EventServiceImplTest.java:33]: Please inject interface

File: sandbox-web/src/test/java/com/exadel/team2/sandbox/service/impl/EventServiceImplTest.java (hunk 1 above)
Comment (VadimBlagiy) [reply sandbox-web/src/test/java/com/exadel/team2/sandbox/service/impl/EventServiceImplTest.java:24]: ок!

File: sandbox-web/src/test/java/com/exadel/team2/sandbox/service/impl/EventServiceImplTest.java (hunk 2 above)
Comment (VadimBlagiy) [reply sandbox-web/src/test/java/com/exadel/team2/sandbox/service/impl/EventServiceImplTest.java:27]: ок

File: sandbox-web/src/test/java/com/exadel/team2/sandbox/service/impl/EventServiceImplTest.java (hunk 3 above)
Comment (VadimBlagiy) [reply sandbox-web/src/test/java/com/exadel/team2/sandbox/service/impl/EventServiceImplTest.java]: done

**Review by ikantsevich — APPROVED** [review]

**Pull request merged**